cf-agent/verify_files.c

```diff
@@ -595,7 +595,7 @@ static PromiseResult RenderTemplateMustache(EvalContext *ctx, const Promise *pp,
     JsonElement *default_template_data = NULL;
     if (!a.template_data)
     {
-        a.template_data = default_template_data = DefaultTemplateData(ctx);
+      a.template_data = default_template_data = DefaultTemplateData(ctx, NULL);
```

Comment:
Oh right, I forgot this one, bogus indentation. I can fix it when merging though. This is more note-to-self.